src/mongo/dbtests/expressiontests.cpp

```diff
@@ -1446,260 +1446,6 @@ class AddToBsonArray {
 
 }  // namespace FieldPath
 
-
```

Comment:
You can ignore this file! Sorry about that. We have a ticket open to delete this: https://jira.mongodb.org/browse/SERVER-21232